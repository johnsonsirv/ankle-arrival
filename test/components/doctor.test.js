/* eslint-disable react/jsx-props-no-spreading */
/* eslint-disable react/react-in-jsx-scope */
/* eslint-disable no-undef */
import Doctor from '../../src/components/doctor';

function setup() {
  const props = {
    firstname: 'Larry',
    lastname: 'Page',
    email: 'example@larry.com',
    city: 'NY',
    username: 'larryp123',
  };

  const enzymeWrapper = shallow(<Doctor doctor={props} />);

  return { props, enzymeWrapper };
}

describe('<Doctor /> rendering', () => {
  const { enzymeWrapper } = setup();
  it('renders correctly', () => {
    expect(enzymeWrapper).toMatchSnapshot();
  });
  it('should require a prop', () => {
    const inputProps = enzymeWrapper.props();
    expect(inputProps).not.toBeNull();
  });
  it('should render 1 <a> to book appointment', () => {
    expect(enzymeWrapper.find('Link').text()).toMatch(/Book Appointment/);
  });
});
