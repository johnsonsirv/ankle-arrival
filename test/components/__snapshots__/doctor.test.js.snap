// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Doctor /> rendering renders correctly 1`] = `ShallowWrapper {}`;
